video-templates/scripts/cf-source-youtube.js

```diff
@@ -0,0 +1,63 @@
+var $ = jQuery,
+    video_container = $('#cf_youtube_overlay_container'),
```

Comment:
@royboy789 for consitency with other Caldera code, please use $video_container I always use $name for a jQuery object in a variable